app/templates/buyers/review_brief_source.html

```diff
@@ -0,0 +1,170 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}{{ brief.title }} [DRAFT] - Digital Marketplace{% endblock %}
+{% block inside_header %}
+<div class="header-title">
+    <a href="#">Digital Marketplace</a>
+    <strong class="phase-tag">BETA</strong>
+</div>
+{% endblock %}
+{% block proposition_header %}
+<div class="header-proposition">
+  <div class="content">
+    <a href="#proposition-links" class="js-header-toggle menu">Menu</a>
+    <nav id="proposition-menu">
+      <ul id="proposition-links">
+        <li>
+          <a href="#">Guidance</a>
+        </li>
+        <li>
+          <a href="#">Help</a>
+        </li>
+        <li>
+          <a href="#">Log in</a>
+        </li>
+      </ul>
+    </nav>
+  </div>
+</div>
+
+{% endblock %}
+{% block breadcrumb %}
+  {%
+    with
+    items = [
+      {
+          "link": "#",
+          "label": "Digital Marketplace"
```

Comment:
I was trying to understand why we were using so much of what is in the base page template. Is this because we wanted to remove URL's so users couldn't navigate away from the page in the iframe? My concern with this page at the moment is that we will need to remember about this other version of the base page when we come to replacing govuk_template with govuk-frontends template.

Have you thought about perhaps placing all the base page template stuff in another page under a `layout` folder with a name of say "no-navigation-links" and then just calling that page from here?

Reply:
I had an idea to share template components between the Buyer FE (where the template for the public page lives) and this page on the Briefs FE, and disable the links afterwards with Javascript (or something). Both apps still need FE work before this can feasibly be done. However this `review_brief_source.html` template is unlikely to be in this state for very long.